run_local_with_dependencies.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-sm --start API_DEFINITION THIRD_PARTY_APPLICATION API_EXAMPLE_MICROSERVICE API_PLATFORM_MICROSERVICE THIRD_PARTY_DEVELOPER_FRONTEND CONTACT_FRONTEND API_PLATFORM_XML_SERVICES
+sm2 --start API_DEFINITION THIRD_PARTY_APPLICATION API_EXAMPLE_MICROSERVICE API_PLATFORM_MICROSERVICE THIRD_PARTY_DEVELOPER_FRONTEND CONTACT_FRONTEND API_PLATFORM_XML_SERVICES
```

Comment:
Not everyone uses sm2 locally I've just aliased it to am

Reply:
We agreed to switch to sm2 in all our run scripts since this is the permanent new name and it saves every new team member setting up an alias.  We should all be using sm2 locally as sm has been deprecated. Anyone still stuck on sm could alias sm2 to sm.

Reply:
Got a question regarding this... in channel